package.json

```diff
@@ -26,7 +26,8 @@
     "office-ui-fabric-react": "^7.183.0",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
-    "react-hot-loader": "^4.13.0"
+    "react-hot-loader": "^4.13.0",
+    "sass": "^1.52.1"
```

Comment:
Hi, @smitsekhadiaa, why this package is added?

Reply:
Actually I was facing some sass error while doing
 `yarn run start`
so I did `yarn add sass` and it was resolved
should I remove it , run and check weather its working properly or not?

Reply:
I will run `gh pr checkout 356` to pull your code into my machine and run `yarn install` then to test if it work. Please wait for a moment.

Reply:
Seems everything works well as before (in fact there is no sass/scss file in Hypercrx at all but just keep it). 

I did some resesarch, `node-sass` is just a faster C++ version of sass but every time I had a  fresh `yarn install` it takes so long to build `node-sass` from scratch (I'm using ARM Mac). Glad to replace it with `sass`.

And I think `sass` is a devdependency, so please `yarn add -D` it to "devdependencies" and make an amend commit then `git push -f` to this branch(PR).

Reply:
hey @tyn1998 I have made the required changes👍

I'll issue the PR for the corresponding changes in the readme also tonight.

Reply:
> so please `yarn add -D` it to "devdependencies"

<img width="1117" alt="image" src="https://user-images.githubusercontent.com/32434520/171438349-fa0e4d42-f237-4173-b1f8-66b4026014f4.png">

have to remove it first~